modules/openshift-developer-cli-odo-known-issues.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-release-notes.adoc
+
+[id="odo-known-issues_{context}"]
+= Known Issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] An active pointer is not pointing to a current active project.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760574[Bug 1760574] A deleted namespace is being listed in `odo project get` command.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760575[Bug 1760575] `odo app delete' removes application components but not services.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760577[Bug 1760577] `odo push` does not delete OpenShift objects when component name is changed.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760578[Bug 1760578] `odo watch` does not fail with error message for git based components but it should.
```

Comment:
fail with error message for git based -> fail with an error message for git-based